### DISCUSSION TITLE
https://greendolphindance.com/2024/11/03/%E8%B4%AD%E7%89%A9%E5%88%86%E4%BA%AB%EF%BC%8810%EF%BC%89/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#31] [General]
# https://greendolphindance.com/2024/11/03/%E8%B4%AD%E7%89%A9%E5%88%86%E4%BA%AB%EF%BC%8810%EF%BC%89/

Back to the rough ground!

https://greendolphindance.com/2024/11/03/%E8%B4%AD%E7%89%A9%E5%88%86%E4%BA%AB%EF%BC%8810%EF%BC%89/

**Comment by Chiyuru** [comment]
> 而如今居然可以不损坏纸张，只擦掉笔迹，科技真的飞速进步啊！

坏了，我上小学的时候就用过这个东西了，年龄差确认（？？（也没有！小时候基本上也还是用蓝色橡皮和胶带，主要因为可擦笔太贵了x

**Reply by greendolphindance** [reply]
what！我很确信我小学时候没有这玩意，可见驰雨老师如此年轻！
